integration_test/api_test/full_node_interface_test.dart

```diff
@@ -123,10 +124,10 @@ Future<void> main() async {
   final testCatCoins = [
     CoinPrototype(
       parentCoinInfo: Puzzlehash.fromHex(
```

Comment:
hardcoded parent coin info is meaningless because there is possible variation in the simulator, should probably replace it with `Puzzlehash.zeros()` to make this apparent.